src/app/zap-templates/zcl/data-model/chip/drlc-cluster.xml

```diff
@@ -87,9 +72,6 @@ limitations under the License.
     <item name="Level3" value="0x04"/>
     <item name="Level4" value="0x05"/>
     <item name="Level5" value="0x06"/>
-    <item name="Emergency" value="0x07"/>
-    <item name="PlannedOutage" value="0x08"/>
-    <item name="ServiceDisconnect" value="0x09"/>
   </enum>
 
   <enum name="HeatingSourceEnum" type="enum8">
```

Comment:
A note: the types of the first two fields in TemperatureControlStruct (not visible in this diff, because not changed in this PR) are incorrect and not wire-compatible with the spec.  Should be `int16s`, no?

But also, is there a reason to not just add the TemperatureDifference type to the type definitions, just like "temperature" was added?  Separate PR for that, of course.